imgui/core.pyx

```diff
@@ -4546,6 +4593,44 @@ cpdef push_style_var(cimgui.ImGuiStyleVar variable, value):
         return True
 
 
+cpdef push_style_color(
+    cimgui.ImGuiCol variable,
+    float r,
+    float g,
+    float b,
+    float a
```

Comment:
It is not in the core lib but we could add default alpha value for users wanting to use the rgb-only tuples.